frontend/src/App.css

```diff
@@ -1,3 +1,49 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+  height: 100%;
```

Comment:
`height: 100%;` won't have any effect unless you fixed an absolute height on html. I guess `height: 100vh;` was the one to keep ;)